operator-common/src/test/java/io/strimzi/operator/common/OperatorMetricsTest.java

```diff
@@ -68,7 +70,7 @@ public void testSuccessfulReconcile(VertxTestContext context)  {
 
         AbstractWatchableStatusedResourceOperator resourceOperator = resourceOperatorWithExistingResource();
 
-        AbstractOperator operator = new AbstractOperator(vertx, "TestResource", resourceOperator, metrics, null) {
+        AbstractOperator operator = new AbstractOperator(vertx, "TestResource", resourceOperator, metrics, Labels.fromMap(emptyMap())) {
```

Comment:
You updated the tests. But I think you should also add some new tests to cover the new feature you are adding.

Reply:
Yeah, that was something I wanted to ask in the morning